### PR TITLE
Fix: failed to fetch loan template

### LINKED ISSUE #747: Failed to fetch Loan template
**Summary:** 

'Failed to fetch Loan template' error shows when creating Loan application with Loan product 'test21'. I am able to create a loan with it in the web browser.

**Steps to reproduce:** 

Create loan application with loan product 'test21'.

**Observed behavior:** 

'Failed to load Loan template' error when product is selected.

**Device and Android version:** 

Momo tablet, Android 4.4.2

**Screenshots:** 

screenshot_2017-08-07-10-48-11

### PR DESCRIPTION
**Pull request by mayank-kgp**
Fixes #747 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by mayank-kgp** [comment]
@therajanmaurya @droidchef Please review it.

File: mifosng-android/src/main/java/com/mifos/objects/templates/loans/LoanTemplate.java [hunk 1]
```diff
@@ -752,7 +752,7 @@ protected LoanTemplate(Parcel in) {
         this.interestRatePerPeriod = (Double) in.readValue(Double.class.getClassLoader());
         this.interestRateFrequencyType = in.readParcelable(InterestRateFrequencyType.class
                 .getClassLoader());
-        this.annualInterestRate = (Integer) in.readValue(Integer.class.getClassLoader());
+        this.annualInterestRate = (Double) in.readValue(Integer.class.getClassLoader());
```
Comment (droidchef) [inline mifosng-android/src/main/java/com/mifos/objects/templates/loans/LoanTemplate.java]: Integer.class.getClassLoader and casting it to Double, Why ?

**Comment by mayank-kgp** [comment]
@droidchef Sorry, my bad. I have corrected that. You can merge it now. :)

**Comment by mayank-kgp** [comment]
@therajanmaurya Please review it.

**Comment by mayank-kgp** [comment]
@droidchef @therajanmaurya Please review. :)

**Comment by himanshu112** [comment]
demo.mifos works fine but the issue continues when u change demo.mifos to your own server.
Has anyone found a solution on this??

**Pull request merged**